SettingsWindow.Designer.cs

```diff
@@ -28,363 +28,429 @@ protected override void Dispose(bool disposing)
-      this.textBoxTrackCnt.Location = new System.Drawing.Point(399, 63);
-      this.textBoxTrackCnt.Name = "textBoxTrackCnt";
-      this.textBoxTrackCnt.Size = new System.Drawing.Size(74, 25);
-      this.textBoxTrackCnt.TabIndex = 7;
-      this.textBoxTrackCnt.Text = "PartyMax";
-      // 
-      // label3
-      // 
-      this.label3.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
-      this.label3.AutoSize = true;
-      this.label3.Font = new System.Drawing.Font("Arial", 11.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
-      this.label3.ForeColor = System.Drawing.Color.FromArgb(((int)(((byte)(226)))), ((int)(((byte)(230)))), ((int)(((byte)(246)))));
-      this.label3.Location = new System.Drawing.Point(373, 66);
-      this.label3.Name = "label3";
-      this.label3.Size = new System.Drawing.Size(20, 17);
-      this.label3.TabIndex = 6;
-      this.label3.Text = "of";
-      // 
-      // textBoxTrackNo
-      // 
-      this.textBoxTrackNo.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
-      this.textBoxTrackNo.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(114)))), ((int)(((byte)(137)))), ((int)(((byte)(218)))));
-      this.textBoxTrackNo.Font = new System.Drawing.Font("Arial", 11.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
-      this.textBoxTrackNo.ForeColor = System.Drawing.Color.FromArgb(((int)(((byte)(226)))), ((int)(((byte)(230)))), ((int)(((byte)(246)))));
-      this.textBoxTrackNo.Location = new System.Drawing.Point(295, 63);
-      this.textBoxTrackNo.Name = "textBoxTrackNo";
-      this.textBoxTrackNo.Size = new System.Drawing.Size(74, 25);
-      this.textBoxTrackNo.TabIndex = 5;
-      this.textBoxTrackNo.Text = "PartyNo";
-      // 
-      // label2
-      // 
-      this.label2.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
-      this.label2.AutoSize = true;
-      this.label2.Font = new System.Drawing.Font("Arial", 11.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
-      this.label2.ForeColor = System.Drawing.Color.FromArgb(((int)(((byte)(226)))), ((int)(((byte)(230)))), ((int)(((byte)(246)))));
-      this.label2.Location = new System.Drawing.Point(276, 66);
-      this.label2.Name = "label2";
-      this.label2.Size = new System.Drawing.Size(13, 17);
-      this.label2.TabIndex = 4;
-      this.label2.Text = "(";
-      // 
-      // textBoxState
-      // 
-      this.textBoxState.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
+            this.textBoxSmallImage = new System.Windows.Forms.TextBox();
```

Comment:
Somehow the indent on this file is wrong, please check your VS settings

Reply:
weird